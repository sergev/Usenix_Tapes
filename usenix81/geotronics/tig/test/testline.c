/*
 * testline.c - test "line" library routine
 */
main()	{
	unsigned	x[5], y[5];
	int		i ;

	movepen( 1000 , 1000 );
	pendown();
	movepen( 1500 , 1000 );
	movepen( 1500 , 1500 );
	movepen( 1000 , 1500 );
	movepen( 1000 , 1000 );
	for ( i = 0 ; i < 5 ; ++ i )
		{
		x[i] = 1000 + 500 * i ;
		y[i] = 2000 + 100 * i ;
		}
	line( x , y , 5 , 4 , 2 );
	for ( i = 0 ; i < 5 ; ++ i )
		y[i] += 1000 ;
	line( x , y , 5 , 0 , 2 );
	for ( i = 0 ; i < 5 ; ++ i )
		y[i] += 1000 ;
	line( x , y , 5 , -4 , 2 );
	}
