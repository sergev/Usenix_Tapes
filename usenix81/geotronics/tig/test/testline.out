MèèDMÜèMÜÜMèÜMèèUMèÐUM¶…M¶–DMùØUM¶ØDMù–UMèÐDMÜ4DMÐ˜UMžMMž^DMá UMž DMá^UMÐ˜DMÄ	üDM¸`	UM†	M†&	DMÉh	UM†h	DMÉ&	UM¸`	DUUMè¸DMÜDMÐ€DMÄ	äDM¸HDUUMè UM¶UM¶fDMù¨UM¶¨DMùfUMè MÜMÐhUMžMž.DMápUMžpDMá.UMÐhMÄ	ÌM¸0UM†åM†öDMÉ8UM†8DMÉöUM¸0U